test/jdk/javax/swing/JInternalFrame/InternalFrameBorderTest.java

```diff
@@ -215,12 +215,22 @@ private static void checkCorners(String cornerLocation) {
             default -> throw new IllegalStateException("Unexpected value: "
                     + cornerLocation);
         }
+
+        boolean isTop = cornerLocation.equals("TOP_LEFT")
+                || cornerLocation.equals("TOP_RIGHT");
+        boolean isLeft = cornerLocation.equals("TOP_LEFT")
+                || cornerLocation.equals("BOTTOM_LEFT");
+
         robot.mouseMove(x, y);
-        if (Color.RED.equals(robot.getPixelColor(x, y))) {
-            createMRIScreenCapture(cornerLocation + "_uiScale"
-                    + uiScale + ".png");
-            errorLog.append("At uiScale: "+ uiScale + ", Red background color"
-                    + " detected at " + cornerLocation +  " corner\n");
+
+        for (int i = 0; i < 5; i++) {
```

Comment:
```suggestion
        for (int i = 0; i < BORDER_THICKNESS; i++) {
```
Is it the intention?

Reply:
Yes, that is correct.